stdlib/email/__init__.pyi

```diff
@@ -1,5 +1,6 @@
 from collections.abc import Callable
 from email.policy import Policy, _MessageT
+from mailbox import Message
```

Comment:
(oops, that's an embarrassing typo. Serves me right for trusting too much in my IDEs auto import. Not that it would have mattered anyway, since mypy doesn't support this pattern even with the right type being used)